meal.go

```diff
@@ -159,19 +162,162 @@ func getMealDataFromID(client HttpClient, id int) (week []MealData, err error) {
 	// 학식 표 순회
 	for i := 0; i < 5; i++ {
 		// 학식을 추출 후 배열에 추가
-		week = append(week, MealData{
+		week = append(week, mealData{
 			Day:  mealTable.Find(fmt.Sprintf(theadSelector, 1, i+2)).Text(),
 			Date: mealTable.Find(fmt.Sprintf(theadSelector, 2, i+3)).Text(),
-			Lunch: Lunch{
-				A: processDietData(mealTable, 1, i+3),
-				B: processDietData(mealTable, 3, i+2),
-				C: processDietData(mealTable, 5, i+2),
+			Lunch: lunch{
+				A: processdietData(mealTable, 1, i+3),
+				B: processdietData(mealTable, 3, i+2),
+				C: processdietData(mealTable, 5, i+2),
 			},
-			Dinner: Dinner{
-				A: processDietData(mealTable, 7, i+3),
+			Dinner: dinner{
+				A: processdietData(mealTable, 7, i+3),
 			},
 		})
 	}
 
 	return
 }
+
+// 제목에서 기간을 추출하기 위한 정규표현식
+var regexDateFromTitle *regexp.Regexp = regexp.MustCompile("(\\d{1,2})\\D{1,2}(\\d{1,2})\\D{1,2}(\\d{1,2})\\D{1,2}(\\d{1,2})\\D?")
+
+// 현재 시간을 기준으로 해당 주의 시간표를 추출한다.
+func getMealDataWithTime(client HttpClient, now time.Time) (week []mealData, err error) {
+	defer WhereInError(&err, "현재 주간 학식")
+
+	weekday := now.Weekday()
+	if weekday < time.Monday || weekday > time.Friday {
+		err = NotFoundError.CreateError(errors.New("주말에는 학식이 제공하지 않습니다."))
```

Comment:
Codacy found an issue: [error strings should not be capitalized or end with punctuation or a newline](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091160)